bambi/priors/scaler_mle.py

```diff
@@ -215,23 +224,26 @@ def scale(self):
             + ["group_specific"] * len(group_specific)
         )
 
+        # Scale response
+        self.scale_response()
+
         # Initialize terms in order
         for term, term_type in zip(terms, term_types):
             # Only scale priors if term or model is set to be auto scaled.
             # By default, use "wide".
+            if not term.prior.auto_scale:
+                continue
+
             if term.prior.scale is None:
-                # pylint: disable=protected-access
-                if not (term.prior._auto_scale and self.model.auto_scale):
-                    continue
                 term.prior.scale = "wide"
 
-            if self.mle is None:
-                self.fit_mle()
-
             # Convert scale names to floats
             if isinstance(term.prior.scale, str):
                 term.prior.scale = self.names[term.prior.scale]
 
+            if self.mle is None:
+                self.fit_mle()
+
             # Scale it
```

Comment:
expand it to full name, makes comment easier to read in isolation

Reply:
What do you mean with full name? Something like `self.fit_maximum_likeliihood_estimator`? Or is it about the comment below?